solr6/local/91-healthcheck.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+echo "Solr init healthcheck start"    
+
+SOLR_DIR_ROOT="${SOLR_INSTALL_HOME}/solrhome"
+ALFRESCO_SSL=${ALFRESCO_SSL:-'https'}
```

Comment:
Aren't these defaults already set somewhere else in a script that runs before this script?
Having to put these defaults in every file is a bit error-prone.
Because all these scripts are `source`d inside an other script, they have access to the variables of scripts that have run earlier.

Reply:
Correct. This is the first script ran, I had removed already (some of) variables redefined in the second script, but not all. Adapted.